# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoConduit.h
# Commit: 8d60ee1f7cd7
# Full Hash: 8d60ee1f7cd7cc655c34d66e070feea356634997
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Remove CODEC_PLNAME_SIZE as CodecInst no longer exists upstream. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102277
# ==============================================================================

diff -r f16cf2573a4d -r 8d60ee1f7cd7 dom/media/webrtc/libwebrtcglue/VideoConduit.h
--- a/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Mon Jan 18 11:16:22 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Fri Jan 29 17:03:42 2021 +0100
@@ -69,9 +69,6 @@
       public rtc::VideoSinkInterface<webrtc::VideoFrame>,
       public rtc::VideoSourceInterface<webrtc::VideoFrame> {
  public:
-  // VoiceEngine defined constant for Payload Name Size.
-  static const unsigned int CODEC_PLNAME_SIZE;
-
   // Returns true when both encoder and decoder are HW accelerated.
   static bool HasH264Hardware();
 
